arex-storage-web-api/src/main/java/com/arextest/storage/web/controller/config/AgentRemoteConfigurationController.java

```diff
@@ -147,13 +145,9 @@ public Response load(@RequestBody AgentRemoteConfigurationRequest request) {
 
       // asynchronously update application env
       asyncUpdateAppEnv(requestInstance);
+      CompareConfiguration compareConfiguration = queryConfigService.queryCompareConfiguration(appId);
```

Comment:
suggest `queryCompareConfiguration` method add try catch, avoid throw exception to affect normal logic.